avocado/core/nrunner.py

```diff
@@ -488,13 +488,51 @@ def _create_params(self):
 
         return params
 
+    @staticmethod
+    def _get_avocado_version():
+        """Return the Avocado package version, if installed"""
+        version = "unknown.unknown"
+        if PKG_RESOURCES_AVAILABLE:
+            try:
+                version = pkg_resources.get_distribution(
+                    'avocado-framework').version
+            except pkg_resources.DistributionNotFound:
+                pass
+        return version
+
+    def get_env_variables(self):
```

Comment:
Maybe make this an "under" method like the `_get_avocado_version()` method above?